package.json

```diff
@@ -16,7 +16,7 @@
     "bootstrap-evergreen": "cross-env ELECTRON_SKIP_BINARY_DOWNLOAD=1 lerna exec -- npm ci --quiet && npm run install-electron",
     "install-electron": "cross-env ELECTRON_SKIP_BINARY_DOWNLOAD= lerna exec --concurrency 1 -- node ../../scripts/install-electron.js",
     "start": "lerna run start --stream --scope mongodb-compass",
-    "amend-compass-version": "node scripts/bump-compass-dependencies.js && git commit --no-allow-empty --amend --no-edit packages/compass/package.json packages/compass/package-lock.json",
+    "amend-compass-version": "node scripts/bump-compass-dependencies.js && git commit --no-allow-empty -m \"chore(release): sync compass dependencies\" packages/compass/package.json",
```

Comment:
We are not updating compass package-lock here, but probably should in the "release" phase, right?

Reply:
yeah the package-lock is not being changed but will be on publish i believe